scipy/optimize/_lbfgsb_py.py

```diff
@@ -344,6 +345,10 @@ def _minimize_lbfgsb(fun, x0, args=(), jac=None, bounds=None,
         possibly adjusted to fit into the bounds. For ``method='3-point'``
         the sign of `h` is ignored. If None (default) then step is selected
         automatically.
+    workers : map-like callable, optional
+        A map-like callable, such as `multiprocessing.Pool.map` for evaluating
+        any numerical differentiation in parallel.
+        This evaluation is carried out as ``workers(fun, iterable)``.
```

Comment:
FWIW, a longer explanation would be nice --- not here of course, maybe a section in a tutorial with a link from here or from the notes section?
The wording here is nice actually: if I as a reader know a bit about the general pattern, that's all the info I need here; if it's my first encounter with the pattern, just having a reference to a longer explanation would be sufficient.

Reply:
A longer demo in a tutorial makes sense.